pkg/api/server/cel2sql/convert_test.go

```diff
@@ -18,265 +18,44 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/tektoncd/results/pkg/api/server/cel"
-
+	"github.com/google/cel-go/cel"
 	"github.com/google/go-cmp/cmp"
 )
 
-func TestConvertRecordExpressions(t *testing.T) {
```

Comment:
As mentioned previously, I recommend keeping these tests here  instead of moving them to a different package.